concourse/tasks/delete-ssh-keys.yml

```diff
@@ -0,0 +1,21 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/awscli
+inputs:
+  - name: paas-cf
+run:
+  path: sh
+  args:
+    - -e
+    - -c
```

Comment:
It'll work because `-u` checks if the variable is set, not if it's contents are empty:
```sh
➜  paas-cf git:(master) ✗ env ./test.sh
./test.sh: line 3: SSH_KEY_PREFIX: unbound variable
➜  paas-cf git:(master) ✗ env SSH_KEY_PREFIX="" ./test.sh
SSH_KEY_PREFIX:
➜  paas-cf git:(master) ✗ env SSH_KEY_PREFIX="foo" ./test.sh
SSH_KEY_PREFIX: foo
```